src/content/docs/build/get-started/ethereum-cheatsheet.mdx

```diff
@@ -5,37 +5,47 @@ sidebar:
-| **Caller ID**              | `msg.sender`                                                                                                                   | `&signer` reference                                                                                                                   |
 | **Upgradeability**         | Proxy patterns                                                                                                                 | Direct module upgrades                                                                                                                |
 | **Safety & Security**      | Vulnerable to attacks like reentrancy                                                                                          | Mitigates common vulnerabilities                                                                                                      |
 | **Dispatch Type**          | Dynamic dispatch                                                                                                               | Static dispatch                                                                                                                       |
-| **FT Standard**            | [ERC-20](https://docs.openzeppelin.com/contracts/4.x/erc20)                                                                    | [Coin](/build/smart-contracts/aptos-coin) (legacy) and [Fungible Asset](/build/smart-contracts/fungible-asset)                        |
+| **Frontend SDK**           | [Ethers.js library](https://docs.ethers.org/v6/)                                                                               | [Aptos Typescript SDK](/build/sdks/ts-sdk)                                                                                            |
 | **NFT Standards**          | [ERC-721](https://docs.openzeppelin.com/contracts/4.x/erc721), [ERC-1155](https://docs.openzeppelin.com/contracts/4.x/erc1155) | [Digital Asset](/build/smart-contracts/digital-asset)                                                                                 |
-| **Blockchain Interaction** | [Ethers.js library](https://docs.ethers.org/v6/)                                                                               | [Aptos Typescript SDK](/build/sdks/ts-sdk)                                                                                            |
+| **FT Standard**            | [ERC-20](https://docs.openzeppelin.com/contracts/4.x/erc20)                                                                    | [Fungible Asset](/build/smart-contracts/fungible-asset) and [Coin](/build/smart-contracts/aptos-coin) (legacy)                         |
+| **Example Code**           | [ERC-20](https://github.com/OpenZeppelin/openzeppelin-contracts/tree/master/contracts/token/ERC20)                             | [Fungible Asset](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/fungible_asset.move) |
```

Comment:
Hmm... associated with this, would be good to point out that you don't have to redeploy the same ERC20 contract with things changed.

Maybe something like... factory pattern vs copy pasta